src/main/webapp/app/admin/metrics/blocks/metrics-modal-threads/metrics-modal-threads.component.html

```diff
@@ -6,39 +6,39 @@ <h4 class="modal-title" jhiTranslate="metrics.jvm.threads.dump.title">Threads du
 
 <div class="modal-body">
     <div class="mb-3">
-        <span class="badge bg-primary hand" (click)="threadStateFilter = undefined">
-            <fa-icon *ngIf="threadStateFilter === undefined" icon="check"></fa-icon>
+        <span class="badge bg-primary hand" (click)="selectedThreadState = undefined">
+            <fa-icon *ngIf="selectedThreadState === undefined" icon="check"></fa-icon>
```

Comment:
minor thing.. if it does not fit this case, you can ignore it :) 
```suggestion
            <fa-icon *ngIf="selectedThreadState == undefined" icon="check"></fa-icon>
```